corehq/apps/receiverwrapper/util.py

```diff
@@ -209,17 +211,23 @@ def get_app_and_build_ids(domain, build_or_app_id):
     return build_or_app_id, None
 
 
-def from_demo_user(form_json):
+def from_demo_user(form_json, domain):
```

Comment:
This also get's called here FYI: https://github.com/dimagi/commcare-hq/blob/7fbe93cb5c6091ab2b11b3186ab49aedfc776a54/corehq/apps/receiverwrapper/views.py#L221

Reply:
https://github.com/dimagi/commcare-hq/pull/25615/commits/48110bc0118926ac35e4bc555a0c0c743618036a